docs/package-sign.md

```diff
@@ -102,7 +102,7 @@ For example: 
 (.venv) D:\connector-plugin-sdk\connector-packager>keytool -genkey -alias test1year -keystore test1yearkeystore.jks -validity 365 
 ```
 
-For more information about keytool arguments, see the Java Documentation about [keytool](https://docs.oracle.com/javase/8/docs/technotes/tools/unix/keytool.html) on the Oracle website. 
+The default password for keystore is changeit. For more information about keytool arguments, see the Java Documentation about [keytool](https://docs.oracle.com/javase/8/docs/technotes/tools/unix/keytool.html) on the Oracle website. 
```

Comment:
Is there any way you can markdown blockquote from the original article about keytool's choice about default password.  I do not want to imply in any way this is Tableau or the SDK's choice, but underlying tooling.  

If we have to include it in our docs because its a common pain point I want it clear the sourcing.

Reply:
I see, understand, my understanding is that maybe I should remove this part preventing any implication. Thanks a lot!

Reply:
I didn't think enough on this in the first place, thanks for the instructions!